docs/release-notes/8.0.42-33.md

```diff
@@ -0,0 +1,115 @@
+
+### MySQL 8.0.42
+
+Improvements and bug fixes provided by Oracle for MySQL 8.0.42 and included in Percona Server for MySQL are the following:
+
+* Fixed an issue where `CHECK TABLE` sometimes incorrectly reported that spatial indexes were corrupted. (Bug #37286473)
+
+* Fixed an issue in InnoDB redo log recovery to improve data safety after a crash. (Bug #37061960)
+
+* Fixed an issue where reading `index_id` values could lead to incorrect behavior with indexes. (Bug #36993445, Bug #37709706)
+
+* Fixed a bug related to the `lower_case_table_names` setting that caused inconsistent behavior with table names on different systems. (Bug #32288105)
+
+* Fixed a bug where `mysqldump` did not properly escape certain special characters in its output. (Bug #37540722, Bug #37709163)
+
+* The `fprintf_string()` function in `mysqldump` did not use the correct quote character for escaping strings. (Bug #37607195)
+
+Find the complete list of bug fixes and changes in the [MySQL 8.0.42 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-42.html).
+
+## Improvements
+
+[PXC-4645](https://perconadev.atlassian.net/browse/PXC-4645): Improves State Snapshot Transfer (SST) failure diagnostics. `garbd` now uses distinct exit codes to differentiate between donor exit, SST script failure, and garbd-initiated termination, making SST issues easier to identify and debug.
+
+## Bug fixes
+
+* [PXC-3576](https://perconadev.atlassian.net/browse/PXC-3576): Percona XtraDB Cluster created role mappings using an upper-case database name when `lower_case_table_names=1` was set.
+
+* [PXC-4394](https://perconadev.atlassian.net/browse/PXC-4394): If compression was enabled for backups and State Snapshot Transfer (SST), the error log showed a misleading version comparison error. This occurred because Percona Xtrabackup was unable to return its version.
+
+* [PXC-4399](https://perconadev.atlassian.net/browse/PXC-4399): Executing `FLUSH TABLES` on a table while writes were in progress caused the cluster node to stall.
+
+* [PXC-4512](https://perconadev.atlassian.net/browse/PXC-4512): When `RENAME TABLE` was executed on a child table concurrently with `INSERT`/`UPDATE` on a parent table, a Metadata Lock (MDL) BF-BF conflict occured on the replica, causing two cluster nodes to enter NON-PRIMARY state.
+
+* [PXC-4629](https://perconadev.atlassian.net/browse/PXC-4629): Creating a temporary table with `CREATE TEMPORARY TABLE ... AS SELECT` put the entire session into Transaction Ordered Index (TOI ) / Next Binary Operation (NBO) lock state, preventing any queries from being killed within that session.
+
+* [PXC-4638](https://perconadev.atlassian.net/browse/PXC-4638): After performing State Snapshot Transfer (SST), the `binlog.index` file contents became inconsistent, causing the `binlog_utils_udf` plugin to fail when accessing the binlog file.
```

Comment:
Actually it was fixed by PS-9777. Maybe worth mentioning?

Reply:
added